src/js/components/explorer/detail/DetailContent.jsx

```diff
@@ -56,16 +56,16 @@ export default class DetailContent extends React.Component {
         this.updateChart(this.props.data);
     }
 
-    componentWillReceiveProps(nextProps) {
-        if (nextProps.transitionSteps !== 0 && nextProps.transition !== this.props.transition) {
-            if (nextProps.transition === 'start') {
-                this.startTransition(nextProps.transitionSteps);
+    componentDidUpdate(prevProps) {
+        if (prevProps.transitionSteps !== 0 && prevProps.transition !== this.props.transition) {
+            if (prevProps.transition === 'start') {
+                this.startTransition(prevProps.transitionSteps);
             }
-            else if (nextProps.transition === 'end') {
+            else if (prevProps.transition === 'end') {
```

Comment:
we should be replacing `nextProps` with `this.props`. `prevProps` should only be used for comparison.